src/main/js/component/general/distribution/Index.js

```diff
@@ -84,18 +84,21 @@ class Index extends Component {
             const {distributionConfigId, distributionType} = currentRowSelected;
             if (distributionType === 'channel_email') {
                 return (<GroupEmailJobConfiguration
+                    alertChannelName={distributionType}
```

Comment:
These all have the same alertChannelName?

Reply:
Similar to the item about the distributionType is job specific and is "channel_email", "channel_hipchat", or "channel_slack" So they aren't the same.  It just looks that way because it's the same variable value.  In the future we won't have a switch statement here.